core/src/main/java/org/axonframework/messaging/Message.java

```diff
@@ -35,7 +36,7 @@
  * @see EventMessage
  * @since 2.0
  */
-public interface Message<T> extends Serializable {
+public interface Message<T> extends Serializable, SerializationAware {
```

Comment:
If all messages are `SerializationAware`, I think we should remove the interface and move the method declarations to `Message`.